src/Transaction.php

```diff
@@ -2,5 +2,98 @@
 
 namespace NewsLadder\PublisherAPI;
 
+use NewsLadder\PublisherAPI\Config;
+use NewsLadder\PublisherAPI\Request;
+
```

Comment:
Within the sub-namespace the use statement can be skipped and the class can be used directly, e.g. `Config` in this case.

```suggestion
```